templates/quarkus-camel-api-consumer/skeleton/pom.xml

```diff
@@ -33,6 +33,7 @@
         <maven-jar-plugin.version>3.4.2</maven-jar-plugin.version>
         <maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
         <maven-surefire-plugin.version>3.4.0</maven-surefire-plugin.version>
+        <registry.version>2.6.4.Final</registry.version>
```

Comment:
The name of this property is slightly ambiguous. To follow the naming conventions of other properties that define plugin versions, I'd recommend using the artifactId of the plugin as part of the property name; ie, `<apicurio-registry-maven-plugin.version>2.6.4.Final</apicurio-registry-maven-plugin.version>`.

Reply:
_registry.version_ comes from apicurio docs, but you are right, let me align the naming convention.